android/build.gradle

```diff
@@ -14,6 +14,9 @@ buildscript {
         googlePlayServicesLocationVersion = "21.0.1"
         mercadoPagoPxVersion = "4.53.2"
         reactNativeVersion = "0.73.5" // https://github.com/expo/expo/issues/18129
+        // https://github.com/proyecto26/react-native-inappbrowser#mostly-automatic-installation
+        androidXAnnotation = "1.2.0"
+        androidXBrowser = "1.3.0"
```

Comment:
@r0xsh can you try on Android with latest versions of those? 
- https://mvnrepository.com/artifact/androidx.annotation/annotation
- https://mvnrepository.com/artifact/androidx.browser/browser